integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1134,8 +1134,7 @@ func (s *DockerNetworkSuite) TestDockerNetworkHostModeUngracefulDaemonRestart(c
 		assert.NilError(c, err, out)
 
 		// verify container has finished starting before killing daemon
-		err = s.d.WaitRun(cName)
-		assert.NilError(c, err)
+		cli.WaitRun(c, cName)
```

Comment:
Guessing this won't work, as this doesn't know what daemon to connect to, so it probably is connecting with the default daemon, not the daemon started for this test 🤔